test/unit/HeartbeatTest.js

```diff
@@ -0,0 +1,36 @@
+// Third-party dependencies
+const { expect, use } = require('chai')
+const { describe, it } = require('mocha')
+const sinonChai = require('sinon-chai')
+
+// In-house dependencies
+const index = require('../../index')
+
+// Configure Chai
+use(sinonChai)
+
+describe('HeartbeatTest.js unit tests', () => {
+  describe('when given state transitions array', () => {
+    it('should reformat the state transitions array', () => {
+      const stateTransitionsArray1 = [[0, 0, 1]]
+      const stateTransitionsArrayReadable1 = [
+        {
+          state: 0,
+          reason: 'movement',
+          time: 1,
+        },
+      ]
+      expect(stateTransitionsArray1.map(index.convertStateArrayToObject)).to.deep.equal(stateTransitionsArrayReadable1)
+
+      const stateTransitionsArray2 = [[1, 4, 4294967295]]
+      const stateTransitionsArrayReadable2 = [
+        {
+          state: 1,
+          reason: 'duration_alert',
+          time: 4294967295,
+        },
+      ]
+      expect(stateTransitionsArray2.map(index.convertStateArrayToObject)).to.deep.equal(stateTransitionsArrayReadable2)
```

Comment:
As a general rule of thumb, there should only ever be one `expect` statement per test. This is because as soon as a test encounters a failure, it will stop executing and report it. So if this test failed the first time it calls `expect` then as the person reading the output, you wouldn't know whether or not it would have passed this second test. Then you would potentially fix the first one and need to re-run the test in order to know if you need to fix anything in the second one or not. So this should be split up into two different `it` blocks. 

That will also add the convenience of you not needing to put numbers at the end of your variable names.